R/house_sales_prediction.R

```diff
@@ -0,0 +1,13 @@
+#' Predicting House Prices
+#'
+#' This dataset contains house sale prices for King County, which includes Seattle.
+#' It includes homes sold between May 2014 and May 2015.
+#' It's a great dataset for evaluating simple regression models.
+#'
+#' @source https://www.kaggle.com/harlfoxem/housesalesprediction
+#' License: CC0: Public Domain
+#' @format A data.table with 19 house features plus the price and the id columns, 21613 observations.
+#''
+#' @examples
+#' data("house_sales_prediction", package = "mlr3book")
+"house_sales_prediction"
```

Comment:
Files should always end with an empty line.

Reply:
FFS, you guys need to setup editorconfig.